lib/identity_cache/cached/prefetcher.rb

```diff
@@ -44,10 +44,9 @@ def fetch_association(klass, association, records)
             return
           end
 
-          cached_association = klass.cached_has_manys[association] ||
-                               klass.cached_has_ones[association] ||
-                               klass.cached_belongs_tos[association] ||
-                               raise(ArgumentError, "Unknown cached association #{association} listed for prefetching")
+          cached_association = klass.send(:cached_association, association) || raise(
```

Comment:
It will actually use Hash#fetch for the last lookup in `cached_association` to raise an error instead of returning `nil`, so this `|| raise` is dead code right now.